platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -64,21 +67,35 @@
 
     private final MBeanServer mBeanServer;
 
+    private final List<ConfigurationMigratable> configurationMigratables;
+
+    private final List<DataMigratable> dataMigratables;
+
     /**
      * Constructor.
      *
      * @param configurationAdminMigration object used to export {@link org.osgi.service.cm.Configuration}
      *                                    objects from {@link org.osgi.service.cm.ConfigurationAdmin}
      * @param mBeanServer                 object used to register this object as an MBean
+     * @param configurationMigratables    list of {@link ConfigurationMigratable} services. Needs
+     *                                    to be kept up-to-date by the client of this class.
```

Comment:
When you inject the migratables, the client doesn't need to keep them up-to-date.  It should be a live list.